SpatialOS/scripts/download_dependencies.sh

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/env bash
+
+set -e -x
+
+cd "$(dirname "$0")"
+source ./utils.sh
+
+TOOLS_DIR="$(pwd)/../tools/${SDK_VERSION}"
+mkdir -p "${TOOLS_DIR}"
+
+retrievePackage() {
+  TYPE=$1
+  PACKAGE=$2
+  TARGETDIR=$3
+
+  pushd "${TOOLS_DIR}"
```

Comment:
You can put this part of entering/exiting the tools dir outside of this method. Just do it only once.

Reply:
done